### PR TITLE
Gnome desktop/gsettings-desktop-schemas

### PR DESCRIPTION
**Pull request by Cogitri**

### CODE REVIEW

**Comment by Ikke** [comment]
Build error:

> \>\>\> ERROR: py-gnome-gconf*: Arch specific binaries found so arch must not be set to "noarch"

Also a warning:

> \>\>\> WARNING: py-gnome: APKBUILD does not run any tests!

**Comment by Cogitri** [comment]
Uh...not quite sure why it works locally and on CI then.

**Comment by Ikke** [comment]
This was meant for #7418, but I see you already fixed that.

**Comment by Cogitri** [comment]
Well, #7418 isn't my PR :P

Thanks for reviewing.

File: community/gnome-desktop/APKBUILD [hunk 1]
```diff
@@ -1,39 +1,41 @@
...
-makedepends="$depends_dev xkeyboard-config itstool libxml2-utils gtk+3.0-dev libseccomp-dev
-	gsettings-desktop-schemas-dev gtk-doc py-six"
+license="GPL-2.0-or-later, LGPL-2.1-or-later"
+depends_dev="eudev-dev iso-codes-dev libseccomp-dev libx11-dev xkeyboard-config
+	gobject-introspection-dev gsettings-desktop-schemas-dev gtk+3.0-dev
+	"
+makedepends="$depends_dev itstool libxml2-utils gtk-doc py-six meson py3-setuptools
+	glib-dev"
+depends="bubblewrap"
+options="!check" # Can't find its own GSettings schemas
```
Comment (Ikke) [inline community/gnome-desktop/APKBUILD]: This should be reported upstream if it hasn't been already

Comment (Cogitri) [reply community/gnome-desktop/APKBUILD]: https://gitlab.gnome.org/GNOME/gnome-desktop/issues/114

**Pull request closed without merging**